### ISSUE TITLE
Does not compile with erlang 19

### ISSUE THREAD

**Issue by aaronjensen** [processone/fast_xml#9]
```
Compiling 8 files (.erl)
src/fxml.erl:51: syntax error before: '/'
src/fxml.erl:72: syntax error before: '/'
src/fxml.erl:82: syntax error before: '/'
src/fxml.erl:93: syntax error before: '/'
src/fxml.erl:107: syntax error before: '/'
src/fxml.erl:132: syntax error before: '/'
src/fxml.erl:141: syntax error before: '/'
src/fxml.erl:153: syntax error before: '/'
src/fxml.erl:162: syntax error before: '/'
src/fxml.erl:177: syntax error before: '/'
src/fxml.erl:191: syntax error before: '/'
src/fxml.erl:203: syntax error before: '/'
src/fxml.erl:214: syntax error before: '/'
src/fxml.erl:225: syntax error before: '/'
src/fxml.erl:239: syntax error before: '/'
src/fxml.erl:258: syntax error before: '/'
src/fxml.erl:270: syntax error before: '/'
src/fxml.erl:293: syntax error before: '/'
src/fxml.erl:320: syntax error before: '/'
src/fxml.erl:334: syntax error before: '/'
src/fxml.erl:345: syntax error before: '/'
src/fxml.erl:366: syntax error before: '/'
src/fxml.erl:384: syntax error before: '/'
```

**Comment by zinid** [comment]
It is already tracked here: https://github.com/processone/ejabberd/issues/1168

**Comment by aaronjensen** [comment]
Great. For what it's worth, it compiles fine if you delete the `/N ::` from each spec.

**Comment by zinid** [comment]
@aaronjensen yes, we know :) This is not a top priority issue so far, will be fixed in next release most likely.

**Comment by prefiks** [comment]
Fixed in commit da9bfcfedbe46c1bbb2531a66113c2d531a78d69
